lib/simp/rake/helpers/assets/rpm_spec/simp6.spec

```diff
@@ -301,7 +301,7 @@ end
 
 if req_file then
   for line in req_file:lines() do
-    valid_line = (line:match("^Requires: ") or line:match("^Obsoletes: ") or line:match("^Provides: "))
+    valid_line = (line:match("^Requires: ") or line:match("^Obsoletes: ") or line:match("^Provides: ") or line:match("^Recommends: "))
```

Comment:
Should we go whole-hog and include the other weak-dep tags like `Suggests` and `Supplements` and `Enhances`?

Reply:
offline answer: no (not yet)